xrpl/utils/sidechain.py

```diff
@@ -0,0 +1,45 @@
+"""Sidechain-related helper util functions."""
+
+from typing import cast
+
+from xrpl.constants import XRPLException
+from xrpl.models import Memo, Payment
+from xrpl.utils.str_conversions import str_to_hex
+
+
+def create_cross_chain_payment(payment: Payment, dest_account: str) -> Payment:
+    """
+    Creates a cross-chain payment transaction.
+
+    Args:
+        payment: The initial payment transaction. If the transaction is signed, then
+            it will need to be re-signed. There must be no more than 2 memos, since one
+            memo is used for the sidechain destination account. The destination must be
+            the sidechain's door account.
```

Comment:
```suggestion
        payment: The initial payment transaction. If the transaction is signed, then
            it will need to be re-signed. There must be no more than 2 memos, since the
            first memo is used for the sidechain destination account. The destination
            must be the sidechain's door account.
```